lizmap/modules/lizmap/classes/lizmapProxy.class.php

```diff
@@ -392,11 +392,18 @@ static public function createVirtualProfile( $repository, $project, $layers, $cr
         elseif($cacheStorageType == 'redis'){
             // CACHE CONTENT INTO REDIS
 
+            $cacheRedisHost = 'localhost';
+            $cacheRedisPort = '6379';
+            if( property_exists($ser, 'cacheRedisHost')
+                $cacheRedisHost = trim($ser->cacheRedisHost);
+            if( property_exists($ser, 'cacheRedisPort')
+                $cacheRedisPort = trim($ser->cacheRedisPort);
+
             // Virtual cache profile parameter
             $cacheParams = array(
                 "driver"=>"redis",
-                "host"=>"localhost",
-                "port"=>"6379",
+                "host"=>$cacheRedisHost,
+                "port"=>$cacheRedisPort,
                 "ttl"=>$cacheExpiration
```

Comment:
a "key_prefix" parameter should be added (optional, string), in order to indicate key_prefix.